web-ui/src/api/notifications.js

```diff
@@ -46,3 +47,34 @@ export const sendEmail = async (subject, content, html, recipients, cookie) => {
     }
   });
 };
+
+export const emailPDLAssignment = async (member, cookie) => {
+  if (member.pdlId && member.lastName && member.firstName && member.workEmail) {
```

Comment:
I wonder if it's okay for this to silently do nothing when these criteria are not met. Maybe it should write an error message to the console.

Reply:
Added.